subworkflows/local/qiime2_ancom.nf

```diff
@@ -29,6 +29,7 @@ workflow QIIME2_ANCOM {
         .combine( ch_taxlevel )
         .set{ ch_for_ancom_tax }
     QIIME2_ANCOM_TAX ( ch_for_ancom_tax )
+    QIIME2_ANCOM_TAX.out.ancom.subscribe { if ( it.baseName[0].toString().startsWith("WARNING") ) log.warn it.baseName[0].toString().replace("WARNING ","QIIME2_ANCOM_TAX: ").replace(" ancom_log", "") }
```

Comment:
Why `.replace(" ancom_log", "")`? I cannot see this string on the message above.

Reply:
That was left from an old attempt at getting it right, removed in fb61558.